plugins/workbench-resources/src/components/Workbench.svelte

```diff
@@ -507,7 +506,10 @@
   </Popup>
   <DatePickerPopup />
 {:else}
-  No client
+  <div class="flex-col-center justify-center h-full flex-grow">
```

Comment:
Why not check on server?

Reply:
I think in future allow inactive employee use account popup for contact with support as example